src/tombstone.ts

```diff
@@ -21,7 +21,7 @@ interface Tombstone extends RoomObject {
      * other resources are undefined when empty.
      * You can use lodash.sum to get the total amount of contents.
      */
-    store: StoreDefinition;
+    store: GenericStore;
```

Comment:
This should rather be:
`store: StoreDefinitionUnlimited;`
And it is the same for Ruin (which is now delivered on master)